Artifacts/windows-vscode/Artifactfile.json

```diff
@@ -2,15 +2,28 @@
     "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2016-11-28/dtlArtifacts.json",
     "title": "Visual Studio Code",
     "publisher": "rchaganti",
-    "description": "Installs Visual Studio Code",
+    "description": "Installs the latest stable version of Visual Studio Code",
     "tags": [
-        "Windows",
-        "VSCode",
-        "Visual Studio"
+      "Windows",
+      "VSCode",
+      "Visual Studio"
     ],
     "iconUri": "https://github.com/Azure/azure-devtestlab/raw/master/Artifacts/windows-vscode/vscode.png",
     "targetOsType": "Windows",
+    "parameters": {
+      "Version": {
+        "type": "string",
+        "displayName": "Version",
+        "description": "Version of Visual Studio Code to install. Valid values are 32-bit or 64-bit.",
+        "defaultValue": "32-bit",
+        "allowEmpty": false,
+        "allowedValues": [
+          "32-bit",
```

Comment:
Would Default be a good option to have?  Essentially, x64 unless the machine is x86 only?  If someone wants a specific version they can choose a specific value?

Reply:
Yes, this was done for backward compatibility. If the artifact is being used in a formula, adding the parameter without a default would break it. So, the safest thing is to use the default. I do have a version of the artifact with "Auto" as an option, in addition to the 32-bit and 64-bit. But again, because of backward compatibility, this is the better approach, I think.